www/leaflet.css

```diff
@@ -21,6 +21,7 @@
 .leaflet-container {
 	overflow: hidden;
 	-ms-touch-action: none;
+	touch-action: none;

```

Comment:
Can you describe in the commit message what this change is there for?

Best use a format where the first line is concise, the second line empty, and the third line contains the longer description.